examples/todoapp/lib/feature_statistics/views/statistics_page.dart

```diff
@@ -19,11 +19,12 @@ class StatisticsPage extends StatelessWidget {
   Widget build(BuildContext context) => Scaffold(
         appBar: AppBar(
           title: Text(context.l10n.todos),
-          actions: const [
-            AppTodoListBulkEditPopupMenuButtonWithDependencies(),
+          actions: [
+            AppTodoListBulkEditPopupMenuButton(),
```

Comment:
Why the component with dependency is removed ?

Reply:
Please get familiar with this documentation 
- https://github.com/Prime-Holding/rx_bloc/blob/master/packages/rx_bloc_cli/doc/golden_tests.md

Reply:
I have removed the `WithDependencies` because I moved the`TodoListBulkEditBlocType` to the top of the widget tree. 
In order to disable the list of todos when one of the actions is selected, I am listening to the loading state of `TodoListBulkEditBlocType` which needs to be initialized before `TodoListBlocType`

Reply:
Make sense